test/support/TestApiConnector.scala

```diff
@@ -24,12 +24,17 @@ package support
 import cats.effect.IO
 
 import uk.gov.nationalarchives.omega.editorial.connectors.ApiConnector
-import uk.gov.nationalarchives.omega.editorial.models.EditSet
-import uk.gov.nationalarchives.omega.editorial.editSets
+import uk.gov.nationalarchives.omega.editorial.models.{ EditSet, EditSetRecord }
+import uk.gov.nationalarchives.omega.editorial.{ editSetRecords, editSets }
 
 object TestApiConnector extends ApiConnector(null, null, null) {
 
   override def getEditSet(id: String): IO[EditSet] =
     IO.pure(editSets.editSet1)
 
+  override def getEditSetRecord(editSetOci: String, recordOci: String): IO[Option[EditSetRecord]] =
+    IO.pure {
```

Comment:
This is at odds with the style above (33); perhaps we should decide on one?

Reply:
Sure, can do. I think we should stick with the curly brackets, as to me it says "this block has not been executed yet" more.

I'll make that change now.

Reply:
Good point: let's use the curly braces everywhere we use `IO.pure`.

Reply:
Ah, so scalafmt rewrites curly braces to be parens if it's on a single line. That's a bit agressive...

Reply:
Actually, it's fine as is. Even clearer.
 
Confirmed.